website/templates/util/render_node.mako

```diff
@@ -78,7 +78,7 @@
                     <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
                     <dd class="log-content">
                         <span data-bind="if:log.anonymous">
-                            <span><em>A user</em></span>
+                           <span data-bind="text: anonymousUserName"></span>
```

Comment:
@sloria You are correct! It's a serious problem. I didn't test it correctly. Now I moved the variable to logfeed context and use parent context of foreach. Here is testing result.
![screen shot 2015-06-30 at 11 39 36 am](https://cloud.githubusercontent.com/assets/5420789/8435031/14573ab4-1f1d-11e5-9680-f923a1c3763f.png)